src/ecs_cost_estimator/core.clj

```diff
@@ -0,0 +1,39 @@
+(ns ecs-cost-estimator.core
+  (:require [clojure.pprint :refer [pprint]]
+            [ecs-cost-estimator.ecs-specs :as ecs]
+            [ecs-cost-estimator.hcl-parser :as parser]))
```

Comment:
This is a rather small codebase, but if it were larger, I'd like to highlight some of the advice I found at [Clojure Do’s: Namespace Aliases](https://stuartsierra.com/2015/05/10/clojure-namespace-aliases).

For example, it just happened to me that, when I saw the call to `parser/parse` at line 33, I tried to locate that namespace in the list of files and didn't find an exact match, but I figured that `hcl-parser`; however, I could see how doing this again in a larger project would make things a bit harder at times, so I think Stuart's advise is onto something. What do you think?